### PR TITLE
feat(componenet/searchform): Adding search form component

### PR DESCRIPTION
**Pull request by jpukg**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/web-slate/simple-google-search/blob/main/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What type of change does this PR introduce?

- [ ] API
- [x] Frontend

## PR Sub Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #44

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

File: e -iq [hunk 1]
```diff
@@ -0,0 +1,294 @@
+[33mcommit 9eddb12ea645edf8b86cec0f7e9b6b4ee006f202[m[33m ([m[1;36mHEAD -> [m[1;32mfeature/search[m[33m)[m
```
Comment (naveenvignesh5) [inline e -iq:1]: Why is this file being added ?

Could you remove it if it is unintentionally added ?

**Review by naveenvignesh5 — CHANGES_REQUESTED** [review]
Could you add the steps for component creation in PR description

File: client/google-search-app/src/app/search-form/search-form.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="parent">
+
+  <div class="center">
+    <img src="assets/images/googlelogo_color_272x92dp.png" alt="Google Search branding logo">
+
+    <input required="required">
+    <button name="google-search">Google Search</button>
```
Comment (ramsunvtech) [inline client/google-search-app/src/app/search-form/search-form.component.html:7]: these text can be localized in future PRs. #62

File: client/google-search-app/src/app/search-form/search-form.component.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.parent {
+  overflow: hidden;
+  position: absolute;
```
Comment (ramsunvtech) [inline client/google-search-app/src/app/search-form/search-form.component.scss:3]: avoid absolute position which is not recommended in next PRs.

**Review by ramsunvtech — APPROVED** [review]

**Review by shangan23 — APPROVED** [review]
Looks good to me except the file which Naveen pointed out to remove.
Approving.

**Review by karthikeyanpa12 — APPROVED** [review]

File: client/google-search-app/src/app/search-form/search-form.component.scss (hunk 3 above)
Comment (jpukg) [reply client/google-search-app/src/app/search-form/search-form.component.scss:3]: ok. what is the alternative. @ramsunvtech can you elaborate which one should be used instead?

File: client/google-search-app/src/app/search-form/search-form.component.html (hunk 2 above)
Comment (jpukg) [reply client/google-search-app/src/app/search-form/search-form.component.html:7]: ok

File: e -iq (hunk 1 above)
Comment (jpukg) [reply e -iq]: done

**Review by naveenvignesh5 — APPROVED** [review]

**Pull request merged**